x/profile/client/cli/query_validator_by_cons_address.go

```diff
@@ -0,0 +1,44 @@
+package cli
+
+import (
+	"context"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/spf13/cast"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/spn/x/profile/types"
+)
+
+func CmdShowValidatorByConsAddress() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "show-validator-by-cons-address [consAddress]",
+		Short: "shows a validatorByConsAddress",
```

Comment:
```suggestion
		Short: "shows a validator from the consensus address used in the chain to launch",
```

That's something I will check as well in the `launch` module as well. We should see if we can make auto-generated description more descriptive for commands